### DISCUSSION TITLE
Ice sheet freshwater forcing data products

### DISCUSSION THREAD

**Discussion by mankoff** [PCMDI/input4MIPs_CVs#40] [Show and tell]
# Table of Contents

1.  [Overview](#org7575d05)
    1.  [Greenland](#org3b24d43)
    2.  [Antarctica](#org6055df4)
    3.  [Iceberg melt spatial distribution](#org54a8afa)
    4.  [Vertical distribution](#org1db0647)
2.  [Details](#orgec8ad98)
    1.  [Greenland runoff](#org189f210)
    2.  [Greenland discharge](#org083ee7d)
    3.  [Greenland iceberg distribution](#orgb358817)
    4.  [Antarctic iceberg distribution](#aq_iceberg)

Hello. I'm helping generate some freshwater forcing products and wanted to describe them here and get feedback on the product: format, units, metadata, etc. There are are several products. I describe them below. If anyone wants to download an alpha version of the dataset, I can share them.

<a id="org7575d05"></a>

# Overview

<a id="org3b24d43"></a>

## Greenland

-   Runoff: This has two sources locations and two sink destination.
    -   Source: Ice runoff. This is rain, melted snow, and melted ice from the ice sheet.
    -   Source: Land runoff: Rain an melted snow from land
    -   Sink: Sub-aerial: This is terrestrial streams that enter the fjord surface
    -   Sink: Subglacial: Discharge at depth from marine-terminating glaciers.
-   Discharge: This is solid ice across flux gates ~5 km upstream of the ice/ocean boundary
    -   At the ice/ocean boundary, this splits into submarine melt and iceberg calving. The split is 50 % +- 40 % (Enderlin (2013) <http://doi.org/10.3189/2013JoG12J049> ).

There are other freshwater terms that may add up to ~10 % of the above, but are not yet included in our products. These include:

-   Grounded ice basal melt
-   GROSS Frontal retreat (not net!)
-   Grounding line retreat
-   Processes within unresolved fjords (sea ice formation and melt, snow- and rain- on-surface, etc.

<a id="org6055df4"></a>

## Antarctica

Similar to above, except discharge is more clearly split into sub-shelf melt and shelf calving.

<a id="org54a8afa"></a>

## Iceberg melt spatial distribution

Some icebergs melt within a fjord in Greenland. For those that escape, and for icebergs in Antarctica, we are generating weighted spatial maps for regions of interest (ROIs).

There are 7 ROIs in Greenland, and 18 in Antarctica. We will provide 1 map with the terrestrial source ROI, and for each of those ROIs one ocean map of where iceberg melt occurs.

I assume the ocean iceberg melt map should sum to 1, or they could sum to 1/cell<sub>area</sub>. Or they could have units kg/s based on the data products we used to generate the spatial melt maps. Or all of the above could be provided - it's just a few more fields in the NetCDF files.

<a id="org1db0647"></a>

## Vertical distribution

We will also recommend how to distribute the freshwater vertically. This includes subglacial discharge, submarine melt, and iceberg meltwater.

<a id="orgec8ad98"></a>

# Details

The work for all of the above is ongoing. The data work is at <https://github.com/NASA-GISS/freshwater-forcing-workshop/> and discussions are at <https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions>

<a id="org189f210"></a>

## Greenland runoff

Workbook:
<https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/greenland_runoff.org>

Output from `ncdump` shows four product (ice to surface or subglacial; land to surface or subglacial). There is no separation of ice, snow, or rain.

<details><summary>Output of `ncdump -h`</summary>

```
netcdf GL_runoff {
dimensions:
	region = 7 ;
	time = 876 ;
variables:
	string region(region) ;
	int64 time(time) ;
		time:long_name = "time of measurement" ;
		time:standard_name = "time" ;
		time:axis = "T" ;
		time:units = "days since 1950-01-01" ;
		time:calendar = "proleptic_gregorian" ;
	double subglacial_discharge_from_land(region, time) ;
		subglacial_discharge_from_land:_FillValue = NaN ;
	double surface_runoff_from_land(region, time) ;
		surface_runoff_from_land:_FillValue = NaN ;
	double subglacial_discharge_from_ice(region, time) ;
		subglacial_discharge_from_ice:_FillValue = NaN ;
	double surface_runoff_from_ice(region, time) ;
		surface_runoff_from_ice:_FillValue = NaN ;
	float min_discharge_depth(region) ;
		min_discharge_depth:_FillValue = NaNf ;
		min_discharge_depth:long_name = "height_above_mean_sea_level" ;
		min_discharge_depth:standard_name = "altitude" ;
		min_discharge_depth:units = "m" ;
		min_discharge_depth:positive = "up" ;
		min_discharge_depth:axis = "Z" ;
}
```
</details>

<a id="org083ee7d"></a>

## Greenland discharge

Workbook: <https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/greenland_discharge.org>

Here there is no separation between submarine melt and calving. That separation (50/50?) will be described in the text, and left for modelers to implement. Advice about how to treat icebergs if fjords are not resolved will also be provided in text not data.

<details><summary>Output of `ncdump -h`</summary>

```
netcdf GL_discharge {
dimensions:
	time = 185 ;
	region = 7 ;
variables:
	float discharge(time, region) ;
		discharge:_FillValue = NaNf ;
		discharge:long_name = "Marine discharge. Includes both calving and submarine melt. Some calvinvg is equivalent to submarine melt if using at fjord mouth" ;
		discharge:standard_name = "land_ice_mass_tranport" ;
		discharge:units = "Gt yr-1" ;
	float err(time, region) ;
		err:_FillValue = NaNf ;
		err:long_name = "Marine mass balance uncertainty" ;
		err:standard_name = "land_ice_mass_tranport" ;
		err:units = "Gt yr-1" ;
	string region(region) ;
		region:long_name = "Mouginot 2019 regions" ;
	int64 time(time) ;
		time:cf_role = "timeseries_id" ;
		time:standard_name = "time" ;
		time:axis = "T" ;
		time:units = "days since 1840-01-01 00:00:00" ;
		time:calendar = "proleptic_gregorian" ;

// global attributes:
		:featureType = "timeSeries" ;
		:title = "Greenland ice sheet mass balance from 1840 through next week" ;
		:summary = "Greenland ice sheet mass balance from 1840 through next week" ;
		:keywords = "Greenland; Mass; Mass balance" ;
		:source = "git commit: 20af941" ;
		:creator_name = "Ken Mankoff" ;
		:creator_email = "kdm@geus.dk" ;
		:creator_url = "http://kenmankoff.com" ;
		:institution = "GEUS" ;
		:references = "10.22008/promice/mass_balance" ;
		:product_version = 1. ;
}
```
</details>

<a id="orgb358817"></a>

## Greenland iceberg distribution

Workbook: <https://github.com/NASA-GISS/freshwater-forcing-workshop/blob/main/greenland_iceberg_melt.org>

![GL_berg_melt](https://github.com/PCMDI/input4MIPs_CVs/assets/145117/e59a82e7-9d74-4402-a860-05adff21199d)

<details><summary>Output of `ncdump -h`</summary>

```
netcdf GL_iceberg_melt_3413 {
dimensions:
	y = 205 ;
	x = 146 ;
variables:
	double NO(y, x) ;
		NO:_FillValue = NaN ;
	double NW(y, x) ;
		NW:_FillValue = NaN ;
	double NE(y, x) ;
		NE:_FillValue = NaN ;
	double CW(y, x) ;
		CW:_FillValue = NaN ;
	double CE(y, x) ;
		CE:_FillValue = NaN ;
	double SW(y, x) ;
		SW:_FillValue = NaN ;
	double SE(y, x) ;
		SE:_FillValue = NaN ;
	double GL(y, x) ;
		GL:_FillValue = NaN ;
	double x(x) ;
		x:_FillValue = NaN ;
		x:long_name = "projection_x_coordinate" ;
		x:units = "meters" ;
	double y(y) ;
		y:_FillValue = NaN ;
		y:long_name = "projection_y_coordinate" ;
		y:units = "meters" ;
	int64 spatial_ref ;
		spatial_ref:crs_wkt = "PROJCS[\"WGS 84 / NSIDC Sea Ice Polar Stereographic North\",GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]],PROJECTION[\"Polar_Stereographic\"],PARAMETER[\"latitude_of_origin\",70],PARAMETER[\"central_meridian\",-45],PARAMETER[\"false_easting\",0],PARAMETER[\"false_northing\",0],UNIT[\"metre\",1,AUTHORITY[\"EPSG\",\"9001\"]],AXIS[\"Easting\",SOUTH],AXIS[\"Northing\",SOUTH],AUTHORITY[\"EPSG\",\"3413\"]]" ;
		spatial_ref:semi_major_axis = 6378137. ;
		spatial_ref:semi_minor_axis = 6356752.31424518 ;
		spatial_ref:inverse_flattening = 298.257223563 ;
		spatial_ref:reference_ellipsoid_name = "WGS 84" ;
		spatial_ref:longitude_of_prime_meridian = 0. ;
		spatial_ref:prime_meridian_name = "Greenwich" ;
		spatial_ref:geographic_crs_name = "WGS 84" ;
		spatial_ref:horizontal_datum_name = "World Geodetic System 1984" ;
		spatial_ref:projected_crs_name = "WGS 84 / NSIDC Sea Ice Polar Stereographic North" ;
		spatial_ref:grid_mapping_name = "polar_stereographic" ;
		spatial_ref:standard_parallel = 70. ;
		spatial_ref:straight_vertical_longitude_from_pole = -45. ;
		spatial_ref:false_easting = 0. ;
		spatial_ref:false_northing = 0. ;
		spatial_ref:spatial_ref = "PROJCS[\"WGS 84 / NSIDC Sea Ice Polar Stereographic North\",GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0,AUTHORITY[\"EPSG\",\"8901\"]],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AUTHORITY[\"EPSG\",\"4326\"]],PROJECTION[\"Polar_Stereographic\"],PARAMETER[\"latitude_of_origin\",70],PARAMETER[\"central_meridian\",-45],PARAMETER[\"false_easting\",0],PARAMETER[\"false_northing\",0],UNIT[\"metre\",1,AUTHORITY[\"EPSG\",\"9001\"]],AXIS[\"Easting\",SOUTH],AXIS[\"Northing\",SOUTH],AUTHORITY[\"EPSG\",\"3413\"]]" ;
}
```

</details>

<a id="aq_iceberg"></a>
## Antarctic iceberg distribution

Antarctic iceberg distribution spatial map can be seen at https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/19#discussioncomment-9276832

**Comment by OlgaSergienko** [comment]
It looks like a good start for a long-term project of quantifying the ice-sheet freshwater discharge. Creating mature data sets with robust estimates of magnitudes and their errors will take significantly more effort than one very talented and dedicated person like Ken could put into it.  It will take more time to test and validate such products before they can be used for MIPs.  For the Southern Hemisphere, there appears to be an overlap with [SOFIA MIP](https://sofiamip.github.io/), which results maybe more informative and relevant to address questions of the magnitudes and impacts of the Antarctic freshwater flux. These results and also SOFIA experience as a MIP could be used to encourage interested communities to organize a similar MIP for Greenland.

**Comment by durack1** [comment]
Some progress has been made in this space - see [Schmidt et al., 2025: Datasets and protocols for including anomalous freshwater from melting ice sheets in climate simulations](https://egusphere.copernicus.org/preprints/2025/egusphere-2025-1940) (egusphere), along with the building data products hosted in Zenodo (see https://doi.org/10.5281/zenodo.14020895)

**Reply by mankoff** [reply]
Thanks for keeping this thread alive. Maybe also of interest is https://github.com/mankoff/sankey . Key graphics included here:

<img width="1735" height="1621" alt="image" src="https://github.com/user-attachments/assets/54bff3f0-8ed0-4872-bd38-1a3f0699d5a9" />

**Reply by durack1** [reply]
@mankoff this is great - nice graphics! I am wondering, wouldn't the "Refreezing" and "Runoff" scale directly, and I suppose ditto for the "Sublimation" - really interesting to see how wildly different the relationships between the numbers are - e.g. Greenland 200 vs 435 vs 60, whereas Antarctica 105 vs 10 vs 235.

Curious that "Evaporation" is identical (5 = 5), considering the markedly different surface areas of Greenland vs Antarctica. So much to learn here!

**Reply by mankoff** [reply]
Refreezing is runoff that doesn't leave the ice sheet. So yes, there is some relationship. I'd expect them to scale year on year intra-sheet, but not have the same relationship inter-sheet. Different SMB regimes. In AQ I expect total melt to go up over time, but only small increases in runoff at first - we don't have the same surface stream network, both days and nights > 0 C, etc. Eventually runoff will increase though. In GL if we enter a phase where more nighttime temps are above 0 in the summer, and the ELA climbs, then maybe runoff goes up and refreezing goes down?

Values are rounded to the nearest 5, with the exception of values greater than 0 and less than 2.5, which are rounded up to 5. The raw data is in the README.org at https://github.com/mankoff/sankey . GL EVAP is 7.43, and AQ EVAP is

```
Grounded with islands:
East       0.6060187163407
West       0.2013515636148
Peninsula  0.6982005019075
sum=1.505570781863

Shelves w/o islands:
EVA shelf_without_islands
East       0.70256713317005
West       0.2422464141299
Peninsula  0.70164322473235
sum=1.6464567720323
```

Which is just the MAR output. I wouldn't trust too many of those decimal places, which is why I round to 5.
 
If you want to see everything that went into the ms, wait a short while, this is in review at J. Glac. Or see `ms.org` at the above URL which is source file for the manuscript text. GitHub has a nice table of contents feature too if you click on the hamburger icon. Or email me and I can share the submitted draft ms if you want to see it sooner in PDF format.

**Comment by mankoff** [comment]
Two recent papers:

+ Schmidt _et al._ 2025, **Datasets and protocols for including anomalous freshwater from melting ice sheets in climate simulations** https://doi.org/10.5194/gmd-18-8333-2025

+ Mankoff _et al._ 2025, **Datasets and protocols for including anomalous freshwater from melting ice sheets in climate simulations** https://www.cambridge.org/core/journals/journal-of-glaciology/article/ice-sheet-mass-flow-and-balance-with-constituent-terms-20102019/CC389C664EDB1CFCCAE211F7E2835854?utm_source=SFMC&utm_medium=email&utm_content=Article&utm_campaign=New%20Cambridge%20Alert%20-%20Articles&WT.mc_id=New%20Cambridge%20Alert%20-%20Articles


<img width="758" height="676" alt="mermaid_AQ_gray" src="https://github.com/user-attachments/assets/fa567946-539c-4276-80f7-36d08145fd42" />
